bundles/org.openhab.binding.lgwebos/README.md

```diff
@@ -314,9 +314,9 @@ Sends a button press event to a WebOS device.
 
 Parameters:
 
-| Name    | Description                                                            |
-|---------|------------------------------------------------------------------------|
-| button  | Can be one of UP, DOWN, LEFT, RIGHT, BACK, DELETE, ENTER, HOME, or OK  |
+| Name    | Description                                                                            |
+|---------|----------------------------------------------------------------------------------------|
+| button  | Can be one of UP, DOWN, LEFT, RIGHT, BACK, EXIT, DELETE, ENTER, ENTER_ALT, HOME, or OK |
```

Comment:
If possible, I think we should document the difference between `ENTER `and `ENTER_ALT`. As I understand it, in technical terms, `ENTER` will use a special URL in context of a remote keyboard subscription:
https://github.com/openhab/openhab-addons/blob/99e309e991beadaae37cee09b03c711f88c7f562/bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVKeyboardInput.java#L97-L99

whereas `ENTER_ALT` will use a different (mouse) API?

If we know when each of these should be used, we could document that. Even if we only know that `ENTER` works for older TV's without knowing exactly what that means.

It could also be different purposes? I could only find this in some documentation:

```
void sendEnter () Send enter key to the current text field.
Related capabilities:
• TextInputControl.Send.Enter

void sendDelete () Send delete event to the current text field.
Related capabilities:
• TextInputControl.Send.Delete
```

So looks like intention is to send ENTER to a focused text field. Is that the same as the new ENTER_ALT? Or is it a more "global" kind of SELECT/ENTER?

I have an LG webOS TV also, so could maybe do some tests tomorrow. I know, more questions than answers. But I think we need a bit more understanding to provide documentation explaining this.

Reply:
Unfortunately, I have no answers to your questions. 
@sprehn as author of the binding could probably explain what he implemented but I have the feeling he is no more participating here.

Reply:
What I can tell you is that the keyboard API is mainly used by the sendText binding action which allows to send/display a message on the TV. I have no idea what is exactly the purpose of ENTER and DELETE in that context, even if I can imagine this might be to validate or erase a previously filled text. I don't understand what could be the usage in the context of the binding. Maybe to erase the popup on TV ?

The mouse API is apparently the API to navigate in the TV system like you do with your TV remote control.

Maybe the way ENTER is handled in the sendButton binding action has simply no usage for us ?

But it can be interesting to check the ENTER value with the sendButton action when using the sendText action. I do not remember if the popup disappears automatically after a delay.

Reply:
Oups, confustion between sendText and showToast ! So in fact, I have no use case in mind.
Maybe the keyboard API is when the OS is asking to fill a text ? Like a username or a password ?
Maybe ENTER as value for sendButton action should have been named ENTER_TEXT.
Worst I believe sendButton should have not handled the keyboard API at all, the action description in the documentation is "Sends a button press event to a WebOS device".
So maybe I should just replace the old handling of ENTER by the new ONE.

Reply:
I am going to rename ENTER_ALT into ENTER_NAV and mention that this matches the remote control button in the center of the navigation group.

Reply:
Done

Reply:
Just quick status from my side: I was able to do something with ENTER and DELETE when on-screen keyboard is displayed with focus in the text field:
- Sending ENTER will remove the keyboard.
- Sending DELETE will delete the last character.

I'm not sure yet if they have any other effects, but at least they do something.

Reply:
Ok, I can add a comment in the documentation for DELETE and update the one for ENTER.

Reply:
Done

Reply:
@jlaur : there is also the OK value, I don't know what is its usage, it does nothing when I send it while an item is selected at screen.